### ISSUE TITLE
apps/lens_blur_filter failing on D3D12 backend on buildbots

### ISSUE THREAD

**Issue by steven-johnson** [halide/Halide#5552]
We recently changed this to run tests using smaller images (which fixed out-of-memory failures on other GPU backends), but we are still seeing failures on the D3D12Compute backend (eg https://buildbot.halide-lang.org/master/#/builders/38/builds/45):

```
21/53 Test  #21: lens_blur_filter ...................Exit code 0xc0000409
***Exception:   4.21 sec
Error: Unable to create the Direct3D 12 buffer (HRESULT=0xffffffff887a0005, object*=0x0)
vvvvv D3D12 Begin Debug Dump vvvvv
D3D12 Device Adapter: NVIDIA GeForce GTX 750 Ti
^^^^^ D3D12  End  Debug Dump ^^^^^
D3D12 HALT !!!
```

This may just be another case of a hopelessly-old GPU that needs upgrading, but it would be good to verify that. Assigning for triage.

(In the meantime, I'm going to selectively disable this test on Windows to avoid the failure.)

**Comment by dsharletg** [comment]
This error means:

```
DXGI_ERROR_DEVICE_REMOVED0x887A0005 | The video card has been physically removed from the system, or a driver upgrade for the video card has occurred. The application should destroy and recreate the device. For help debugging the problem, call ID3D10Device::GetDeviceRemovedReason.
```
https://docs.microsoft.com/en-us/windows/win32/direct3ddxgi/dxgi-error

Not sure if that makes sense in this context... maybe it just means this machine needs to be rebooted after an udpate?

**Comment by steven-johnson** [comment]
Both the winbots have been rebooted within the past couple of days, but I can try again.

**Comment by steven-johnson** [comment]
Also, that would suggest that ~all of the GPU tests are failing, but only this one seems to be.

**Comment by shoaibkamil** [comment]
I ran this app locally in a loop for 24hrs and was unable to replicate on my machine with a GTX970.  It may be the old 750ti causing it, but not sure.

**Comment by slomp** [comment]
Yeah, I'm pretty sure it's the curse of the GTX 750 Ti (or its driver) once again 😃

**Comment by abadams** [comment]
> [!NOTE]
> **Automated triage:** this issue was flagged by Claude as likely safe to close, for the reason below. The reasoning was checked against the current source, and a second pass tried and failed to refute it, but no human verified this specific text. **If the reason is wrong, please reopen the issue and say so** — some fraction of these are expected to be wrong.

**Reason (CLOSE_STALE):** This was a `DXGI_ERROR_DEVICE_REMOVED` seen only on one specific old buildbot GPU (GTX 750 Ti); it never reproduced elsewhere, and the thread concluded it was the machine/driver. That buildbot fleet no longer exists (CI is GitHub Actions now), and the Windows-specific disable mentioned in the report is long gone from the app.

**Evidence:** `apps/lens_blur/CMakeLists.txt:31-60` registers the `lens_blur_filter` test unconditionally except for a Metal-only threadgroup-size skip; there is no D3D12/Windows exclusion left. Comments from @shoaibkamil and @slomp in the thread attribute the failure to the GTX 750 Ti.

Closing: this was a one-off `DXGI_ERROR_DEVICE_REMOVED` on the old GTX 750 Ti buildbot, which no longer exists, and `apps/lens_blur` no longer carries any Windows/D3D12 exclusion. Please reopen with a fresh log if `lens_blur` fails on D3D12 on current hardware.
